README.md

```diff
@@ -282,4 +282,4 @@ Although this script is using `subprocess.run(shell=True)` the only possibility
 
 
 
-![quadhelion engineering logo](images/quadhelionEngineeringIcon.jpg)
\ No newline at end of file
+![quadhelion engineering logo](images/quadhelionEngineeringIcon.jpg)
```

Comment:
I don't recall touching this line, and I can't see a difference.

Reply:
I'd really not like to get into the GitHub contributor/merging process as I don't like how GitHub handles that and many other things. I tried to disable it but I don't see a setting for it. 

That is a very nice suggestion, thank you, I appreciate that. I updated it. 

Just open up an issue next time if would you please and again: Thanks @grahamperrin!

Reply:
Thank you, 

> That is a very nice suggestion, thank you, I appreciate that. I updated it.

I don't see the manual page link changed at <https://github.com/wravoc/harden-freebsd/commits/main/README.md> … is the update elsewhere, not yet in this repo? 

(I'm not impatient, just curious.)

Reply:
GH seems to be very slow on the uptake but it went in as I see it, try a forced browser refresh.

Reply:
Thanks, I force-refreshed a few times earlier, the commit is with us now: https://github.com/wravoc/harden-freebsd/commit/fc9d872e2a34e617a6af1bf87afd8ef349dbc696#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5